engine/common/oncall_gateway/oncall_gateway_client.py

```diff
@@ -71,8 +71,8 @@ def post_slack_connector(self, slack_id: str, backend: str) -> tuple[SlackConnec
         response = self._post(url=self._slack_connectors_url, json=d)
         response_data = response.json()
         return (
-            OnCallConnector(
-                response_data["oncall_org_id"],
```

Comment:
what is this change related to?

Reply:
It's just a small bugfix related to the interaction betwenn oncall and oncall-gateway